spec/spec_helper.rb

```diff
@@ -1,5 +1,7 @@
-require 'coveralls'
-Coveralls.wear_merged! 'rails'
+require 'codeclimate-test-reporter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.